pkg/jsonlog/jsonlog.go

```diff
@@ -6,12 +6,20 @@ import (
 	"time"
 )
 
+// JSONLog decsribes a JSONLog.
```

Comment:
JSONLog represents a log message, typically a single entry from a given log stream.
JSONLogs can be easily serialized to and from JSON and support custom formatting.